docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+# Cosmos Hub (Gaia) LCD API
+
+This document describes the API that is exposed by the specific LCD implementation of the Cosmos
+Hub (Gaia). Those APIs are exposed by a REST server and can easily be accessed over HTTP/WS
+connections. 
+
+The complete API is comprised of the sub-APIs of different modules. The modules in the Cosmos Hub
+(Gaia) API are:
+    * ICS0 (TendermintAPI)
+    * ICS1 (KeyAPI)
+    * ICS20 (TokenAPI)
```

Comment:
where are these numbers from?

Reply:
ICS20 to follow the convention that ERC20 has established for Ethereum. It gives people a good mental bridge especially if English isn't there native language. ICS0 and ICS1 come from the fact that I thought we should start at 0.